statsmodels/tsa/arima_model.py

```diff
@@ -925,7 +927,10 @@ def fit(self, start_params=None, trend='c', method="css-mle",
             self.nobs = len(self.endog) - k_ar
 
         if start_params is not None:
-            start_params = np.asarray(start_params)
+            if hasattr(start_params, '__call__'):
```

Comment:
We are also not unit testing python 3.2, and whether statsmodels works with it is doubtful, and not guaranteed.